edb/lib/ext/graphql.edgeql

```diff
@@ -21,19 +21,6 @@ CREATE EXTENSION PACKAGE graphql VERSION '1.0';
 
 CREATE MODULE stdgraphql;
```

Comment:
Can we get rid of this module altogether?

Reply:
We have a helper function in there that converts the names of types into their GraphQL names. It's something that can be inlined I suppose. Just a bit messy.

Reply:
Having a top-level stdlib module just for a helper function is messy too.  As a bonus, you'd be able to actually handle hypothetical cases where `default::` isn't your default module more easily that way.

Reply:
OK, I'll go inline it. maybe it won't be too messy.

Reply:
Gaaah! The short answer is that inlining it breaks `test_graphql_functional_alias_04`. I'm going to need to examine this a bit more thoroughly, because currently there's at least 2 different ways it gets broken: can't prove cardinality ONE for the typename (in the alias computed link) and also getting the wrong result for the type name. They are probably related, but it's a bit messy, so I'm trying to minimize the offending expression.

Reply:
> can't prove cardinality ONE for the typename

You can use the new "assert_single()" for that